adal/build/LayoutNugetPackage.proj

```diff
@@ -24,10 +24,9 @@
   <Import Project="$(MSBuildThisFileDirectory)AssemblySigningVerification.targets"/>
   
   <PropertyGroup>
-    <Configuration Condition="$(Configuration)==''" >Debug</Configuration>
-    
+    <Configuration Condition="$(Configuration)==''" >Release</Configuration>
     <NuspecFilePath Condition="$(NuspecFilePath)==''">$(MSBuildThisFileDirectory)Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec</NuspecFilePath>
-    <AdalBinaryFolder Condition="$(AdalBinaryFolder)==''">..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\netstandard1.1\</AdalBinaryFolder>
+    <AdalBinaryFolder Condition="$(AdalBinaryFolder)==''">$(MSBuildThisFileDirectory)..\src\Microsoft.IdentityModel.Clients.ActiveDirectory.Platform\bin\$(Configuration)\netstandard1.1\</AdalBinaryFolder>
```

Comment:
In previous commit this was $..\ which did not make sense for msbuild

Reply:
cool...thanks.